tasks/make-rootfs/run.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+SUFFIX="${STACKS_SUFFIX:-}"
+
+src-ci/concourse/tasks/common/start-docker.sh
+
+make -C src
+
+versioned_stack_filename="out/opensusefs2${SUFFIX}-$(cat version/number).tar.gz"
+mv src/opensusefs2.tar.gz "${versioned_stack_filename}"
+
+versioned_receipt_filename="out/opensusefs2${SUFFIX}-$(cat version/number).receipt.txt"
+echo "Rootfs SHASUM: $(sha1sum "$versioned_stack_filename" | awk '{print $1}')" > "$versioned_receipt_filename"
+echo "" >> "$versioned_receipt_filename"
```

Comment:
Is this literally just to get a newline in the file? `\n` on the end with `-e` doesn't work? Might break other things. Anyway, just felt weird.